src/vivarium_gates_mncnh/constants/data_values.py

```diff
@@ -289,21 +290,22 @@ class __DeliveryFacilityTypes(NamedTuple):
 # Probability each of these facility types has access to CPAP
 CPAP_ACCESS_PROBABILITIES = {
     "Ethiopia": {
-        NO_CPAP_RISK.P_CPAP_BEMONC: 0.075,
-        NO_CPAP_RISK.P_CPAP_CEMONC: 0.393,
-        NO_CPAP_RISK.P_CPAP_HOME: 0.0,
+        NO_CPAP_RISK.P_CPAP_BEMONC: get_norm(0.075, 0.02**2),
+        NO_CPAP_RISK.P_CPAP_CEMONC: get_norm(0.393, 0.05**2),
+        NO_CPAP_RISK.P_CPAP_HOME: get_norm(0.0, 0.00**2),
     },
     "Nigeria": {
-        NO_CPAP_RISK.P_CPAP_BEMONC: 0.075,
-        NO_CPAP_RISK.P_CPAP_CEMONC: 0.393,
-        NO_CPAP_RISK.P_CPAP_HOME: 0.0,
+        NO_CPAP_RISK.P_CPAP_BEMONC: get_norm(0.075, 0.02**2),
+        NO_CPAP_RISK.P_CPAP_CEMONC: get_norm(0.393, 0.05**2),
+        NO_CPAP_RISK.P_CPAP_HOME: get_norm(0.0, 0.00**2),
     },
     "Pakistan": {
-        NO_CPAP_RISK.P_CPAP_BEMONC: 0.075,
-        NO_CPAP_RISK.P_CPAP_CEMONC: 0.393,
-        NO_CPAP_RISK.P_CPAP_HOME: 0.0,
+        NO_CPAP_RISK.P_CPAP_BEMONC: get_norm(0.075, 0.02**2),
+        NO_CPAP_RISK.P_CPAP_CEMONC: get_norm(0.393, 0.05**2),
+        NO_CPAP_RISK.P_CPAP_HOME: get_norm(0.0, 0.00**2),
     },
 }
+CPAP_RELATIVE_RISK_DISTRIBUTION = get_lognorm_from_quantiles((0.83 + 0.34) / 2, 0.34, 0.83)
```

Comment:
```suggestion
CPAP_RELATIVE_RISK_DISTRIBUTION = get_lognorm_from_quantiles(0.53, 0.34, 0.83)
```

This will give us the correct/expected distribution based on my tests at the bottom of this notebook! https://github.com/ihmeuw/vivarium_research_mncnh_portfolio/blob/main/verification_and_validation/model_7.1.1_nn_checks.ipynb

Reply:
The numbers in Ali's proposal are different. Have we made this change, and why were they different?

Reply:
It's just changing the mean to the mean in the docs rather than the midpoint between the uncertainty interval. The mean should be the mean rather than the midpoint. https://vivarium-research.readthedocs.io/en/latest/models/intervention_models/neonatal/cpap_intervention.html#id7

Reply:
V&V summary and model request plan for updating it are here: https://github.com/ihmeuw/vivarium_research/pull/1669